pkg/email/send_email.go

```diff
@@ -15,6 +16,7 @@ type Emailer struct {
 }
 
 var ErrRateLimitExceeded = errors.New("rate limit exceeded")
+var ErrNoSMTPHostConfigured = errors.New("no smtp host configured")
```

Comment:
Prefer using scope while defining multiple variables or constants
```suggestion
var (
    ErrRateLimitExceeded = errors.New("rate limit exceeded")
    ErrNoSMTPHostConfigured = errors.New("no smtp host configured")
)
```

Reply:
Done